api/managementpb/ia/pagination.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package ia.v1beta1;
+
+option go_package = "api/managementpb/ia;iav1beta1";
+
+// PaginationRequest represents request values for page by page pagination.
+message PaginationRequest {
+  // Maximum number of results per page.
+  int32 page_size = 1;
+  // Index of the requested page, starts from 0.
+  int32 page = 2;
+}
+
+// PaginationResponse represents response values for page by page pagination.
+message PaginationResponse {
+  // Total number of results.
+  int32 total_items = 1;
+  // Total number of pages.
+  int32 total_pages = 2;
+}
```

Comment:
How client should understand what page index to pass to the next request? Calculate it each time form `page_size` and total returned items?

Reply:
No need to calculate, the user chooses the page he wants to go to and this is a page index that passed in the request. 
The FE part knows the number of pages available from the response, so the appropriate number of page links is shown.

Reply:
Ok, thanks for the explanation!